docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>Will those show up in the docs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>